### PR TITLE
feat(performance): added frontend option to display transaction summary page with interpolation and without zerofill

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
The pr includes frontend changes to support displaying transaction summary charts without zerofill and with interpolation
The backend work for this change is in the following pr: https://github.com/getsentry/sentry/pull/27531

Without interpolation:image
With interpolation:image

### CODE REVIEW

**Review by dashed — APPROVED** [review]

File: static/app/components/charts/eventsRequest.tsx [hunk 1]
```diff
@@ -395,9 +400,15 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
       // Convert the timeseries data into a multi-series result set.
       // As the server will have replied with a map like:
       // {[titleString: string]: EventsStats}
+      let timeframe: {start: string; end: string} | undefined = undefined;
       const results: MultiSeriesResults = Object.keys(timeseriesData)
+        .filter((seriesName: string) => !['start', 'end'].includes(seriesName))
```
Comment (Zylphrex) [inline static/app/components/charts/eventsRequest.tsx:405]: Is this still necessary now that the start/end is attached to each individual response?

File: static/app/types/index.tsx [hunk 2]
```diff
@@ -453,10 +453,17 @@ export type EventsStats = {
...
+  end?: string;
 };
 
 // API response format for multiple series
-export type MultiSeriesEventsStats = {[seriesName: string]: EventsStats};
+export type MultiSeriesEventsStats = {
+  [seriesName: string]: EventsStats;
+} & {
+  start?: string;
+  end?: string;
```
Comment (Zylphrex) [inline static/app/types/index.tsx:465]: start/end should only exist on `EventsStats` now right?

File: tests/js/spec/components/charts/eventsRequest.spec.jsx [hunk 3]
```diff
@@ -517,4 +517,35 @@ describe('EventsRequest', function () {
...
+      doEventsRequest.mockClear();
+    });
+
+    it('passes query timeframe start and end to the child if supplied by timeseriesData', async function () {
+      doEventsRequest.mockImplementation(() =>
+        Promise.resolve({
+          p95: {
+            data: [[new Date(), [COUNT_OBJ]]],
+            start: 1,
+            end: 2,
```
Comment (Zylphrex) [inline tests/js/spec/components/charts/eventsRequest.spec.jsx:532]: start/end is declared to be a string above, can we either this a string, or correct the response to be a number?

File: static/app/components/charts/eventsRequest.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/components/charts/eventsRequest.tsx]: 👍  no longer necessary

File: static/app/types/index.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/types/index.tsx]: yup, removed from `MultiSeriesEventsStats`!

File: tests/js/spec/components/charts/eventsRequest.spec.jsx (hunk 3 above)
Comment (edwardgou-sentry) [reply tests/js/spec/components/charts/eventsRequest.spec.jsx]: I've updated `start` and `end` to be `number` types now to support the epoch time change from the backend! Updated the numbers in this test to be a little more realistic as well.

**Pull request merged**